src/components/Image/Image.js

```diff
@@ -43,34 +56,43 @@ class Image extends CloudinaryComponent {
     const {nonCloudinaryProps} = extractCloudinaryProps(options);
 
     let attributes = {...getImageTag(options).attributes(), ...nonCloudinaryProps};
+    attributes = Util.withCamelCaseKeys(attributes);
 
     // Set placeholder Id
     if (placeholder && attributes.id) {
       attributes.id = attributes.id + '-cld-placeholder';
     }
 
+    // Set dataSrc if lazy loading and not in view
+    if (!isInView && this.shouldLazyLoad(options)) {
+      attributes.dataSrc = getConfiguredCloudinary(options).url(options.publicId, options);
+      delete attributes.src;
+    }
+
+    if (attributes.dataSrc) {
+      attributes['data-src'] = attributes.dataSrc;
+    }
```

Comment:
???

Reply:
Looking at this again, calculating a new src isn't required, i've updated this part, please see updated commit